_benchmark/installing-benchmark.md

```diff
@@ -0,0 +1,154 @@
+
+Before you being installing OpenSearch Benchmark, check the following software dependencies.
+
+Use [pyenv](https://github.com/pyenv/pyenv) to manage multiple versions of Python on your host. This is especially useful if your "system" version of Python is older than version 3.8.
+{: .tip}
+
+- Check that Python 3.8 or newer is installed:
+
+  ```bash
+  python3 --version
+  ```
+  {% include copy.html %}
+
+- Check that `pip` is installed and functions:
+
+  ```bash
+  pip --version
+  ```
+  {% include copy.html %}
+
+- _Optional_ Check that your installed version of `git` is **Git 1.9 or newer**. `git` is not required for OpenSearch Benchmark installation, but it is required to fetch benchmark workload resources from a repository when you want to perform tests. See the official Git [Documentation](https://git-scm.com/doc) for help installing Git. 
+
+  ```bash
+  git --version
+  ```
+  {% include copy.html %}
+
+### Completing the installation
+
+After the prerequisites are installed, you can install OpenSearch Benchmark using the following command:
+
+```bash
+pip install opensearch-benchmark
+```
+{% include copy.html %}
+
+After the installation completes you can use the following command to display help information:
+
+```bash
+opensearch-benchmark -h
+```
+{% include copy.html %}
+
+OpenSearch Benchmark will exit with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
+{: .warning}
+
+Now that OpenSearch Benchmark is installed on your host you can learn about [Configuring OpenSearch Benchmark]({{site.url}}{{site.baseurl}}/benchmark/configuring-benchmark/).
+
+## Installing with Docker
+
+You can find the official Docker images for OpenSearch Benchmark on [Docker Hub](https://hub.docker.com/r/opensearchproject/opensearch-benchmark) or on the [Amazon ECR Public Gallery](https://gallery.ecr.aws/opensearchproject/opensearch-benchmark).
+
+To pull the image from Docker Hub:
+
+```bash
+docker pull opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+To pull the image from Amazon ECR:
+
+```bash
+docker pull public.ecr.aws/opensearchproject/opensearch-benchmark:latest
+```
+{% include copy.html %}
+
+### Running Benchmark with Docker
+
+To run OpenSearch Benchmark, you use `docker run` to launch a container. OpenSearch Benchmark subcommands are passed as arguments when you start the container. OpenSearch Benchmark then processes the command, then stops the container after the requested operation completes.
+
+For example, the following command will print the help text for OpenSearch Benchmark to the command line, and then stop the container:
+
+```bash
+docker run opensearchproject/opensearch-benchmark -h
+```
+{% include copy.html %}
+
+OpenSearch Benchmark will exit with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
```

Comment:
```suggestion
OpenSearch Benchmark exits with an error if no arguments are supplied. Use the `-h` option to print the help text in the command line if you want to test the installation.
```

Reply:
Might be clearer to say, "If you want to test the installation, use the 'h' option to display the help text in the command line."